index.html

```diff
@@ -1,259 +1,252 @@
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="button" class="get">Get my resume</button>
+                    <ul>
                         <li>
-                            <a href="#">
-                                <img src="img/Vector1.png" alt="Gmail picture">
-                            </a>
+                            <div class="languages-skills">
+                                <img src="img/Frame 7.png">
+                            </div>
                         </li>
-                    </ul>
-                    <button class="btn-resume">Get my resume</button>
-                </div>
-                <div class="tool">
-                    <button class="language-button">Languages <span class="fa fa-chevron-down"></span></button>
-                    <ul class="languages">
                         <li>
-                            <a href="#"><img src="img/html.png" alt="not found"/><span>html</span></a>
+                            <div class="my-skills">
+                                <img src="img/Frame 9.png" alt="javascript picture">
+                                <img src="img/Frame 10.png" alt="html picture">
+                                <img src="img/Frame 11.png" alt="css picture">
+                            </div>
                         </li>
                         <li>
-                            <a href="#"><img src="img/css.png" alt="not found"/><span>css</span></a>
+                            <div class="languages-skills">
+                                <img src="img/Frame 8.png">
+                            </div>
                         </li>
                         <li>
-                            <a href="#"><img src="img/js.png" alt="not found"/><span>javascript</span></a>
+                            <div class="languages-skills">
+                                <img src="img/Frame 9 (1).png">
+                            </div>
                         </li>
                     </ul>
-                    <button>frameworks <span class="fa fa-chevron-right"></span></button>
-                    <button>Skills <span class="fa fa-chevron-right"></span></button>
-                </div>
-            </div>
+                </section>
         </section>
 
-        <footer>
-            <section class="contact">
-                <div class="contact-container">
-
-                    <h2>Contact me</h2>
-                    <p>
-                        If you have an application you are interested in developing, a feature
-                        that you need built or a project that needs coding. I’d love to help
-                        with it.
-                    </p>
-                    <form action="https://formspree.io/f/mrgjrqoe" method="POST">
-                        <input type="test" name="name" placeholder="Full name" maxlength="30" required />
```

Comment:
- [ ] Kindly fix the typo on your input as the type attribute is supposed to be `text` and not `test` for validation purposes